fhir-apis/patient-api-healthcare-accelerator/service.bal

```diff
@@ -63,6 +63,7 @@ service / on new http:Listener(9090) {
 
         if process is error {
             log:printError("Error in source connection processing");
```

Comment:
As suggested by Ballerina we can add stacktrace of the error to get more information regarding the error.
```suggestion
            log:printError("Error in source connection processing", stacktrace = process.stacktrace());
```